Bottom: 38dcb64a316019d564ed9277cc74dc6c59d92b43
Top:    c5875e0c213201c62216f7538acde1cdfe2355ad
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 13:35:08 -0400

Refresh of stac9766

---

diff --git a/sound/soc/codecs/stac9766.c b/sound/soc/codecs/stac9766.c
index 063bc18..75a3932 100644
--- a/sound/soc/codecs/stac9766.c
+++ b/sound/soc/codecs/stac9766.c
@@ -354,7 +354,7 @@ static struct snd_soc_dai_ops stac9766_dai_ops_digital = {
 	.prepare	= ac97_digital_prepare,
 };
 
-static struct snd_soc_dai stac9766_dai[] = {
+struct snd_soc_dai stac9766_dai[] = {
 {
 	.name	= "stac9766 analog",
 	.id	= 0,
